dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,183 @@
+          }
+          stage('scale down') {
+            // if there is an existing deployment, scale it down
+            openshiftScale(deploymentConfig: 'nodejs-mongodb-example', replicaCount: '0')
+          }
+          stage('build') {
+            // build the application based on the nodejs-mongodb-example build config
+            openshiftBuild(buildConfig: 'nodejs-mongodb-example', showBuildLogs: 'true')
+          }
+          stage('verify build') {
+            // verify that the build completed successfully
+            openshiftVerifyBuild(buildConfig: 'nodejs-mongodb-example')
+          }
+          stage('deploy') {
+            // deploy the application based on the nodejs-mongodb-example deployment config
+            openshiftDeploy(deploymentConfig: 'nodejs-mongodb-example')
+          }
+          stage('verify deployment') {
+            // verify that the application was deployed successfully
+            openshiftVerifyDeployment(deploymentConfig: 'nodejs-mongodb-example', replicaCount: '1')
+          }
+          stage('scale up') {
+            // scale the application up to one (1) pod
+            openshiftScale(deploymentConfig: 'nodejs-mongodb-example', replicaCount: '1')
+          }
+          stage('verify database service') {
+            // verify that the mongodb database service is up and running
+            openshiftVerifyService(serviceName: 'mongodb')
+          }
+          stage('verify http service') {
+            // verify that the node.js http service is up and running
+            openshiftVerifyService(serviceName: 'nodejs-mongodb-example')
+          }
+        }
+   }
+} catch (err) {
+   echo "in catch block"
+   echo "Caught: ${err}"
+   currentBuild.result = 'FAILURE'
+   throw err
+}
+----
+
+Now we can import the build config into our OpenShift cluster with the following command:
+
+[source]
+----
+$ oc create -f nodejs-sample-pipeline.yaml
+----
+
+If you don't want to create your own file, you can use the sample from the Origin repository with the following command:
+
+[source]
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/pipeline/nodejs-sample-pipeline.yaml
+----
+
+For more information about the OpenShift DSL syntax used here see https://jenkins.io/doc/pipeline/steps/openshift-pipeline/[OpenShift Pipeline Steps].
+
+[[starting-the-pipeline]]
+
+== Starting the Pipeline
+
+We can now start the pipeline with the following command:
+
+[source]
+----
+$ oc start-build nodejs-sample-pipeline
+----
+
+The best way to visualize the pipeline execution is by viewing it in the OpenShift web console.  You can view your
+pipelines by logging into the web console and navigating to Builds -> Pipelines.
+
+There you should see the sample-nodejs-pipeline that we created, and each stage that we defined in the Jenkinsfile as it is executed.
+The stages will be green with a check mark if they complete successfully or red with an x if they failed.
```

Comment:
needs some wrap up paragraph saying what should happen if it all works.  (builds the app from X, deploys it, tests it, you now have a running copy of your code at location Y, etc).  

description subject to change per my other comments on your other PR about what i think the content of the jenkinsfile should actually be.